### DISCUSSION TITLE
Unable to connect Jenkins Instance with DTrack using API Key: error 401

### DISCUSSION THREAD

**Discussion by Adedamola86** [DependencyTrack/dependency-track#2148] [General]
Hello All,

For some reason the API key generated via the Frontend (adequate permission, shown in image) seems to fail when I try to configure Jenkin. This fails during test connection with 401 unauthorized. 

I need the SBOMs generated from Jenkins uploaded to DT. Not sure if these are the right keys to use.
Following this instructions: "https://plugins.jenkins.io/dependency-track/"

![DTrack-APIKey](https://user-images.githubusercontent.com/35939196/201502330-26217df0-5787-400f-ac0e-b4b811744ea3.png)
![jenkins-config](https://user-images.githubusercontent.com/35939196/201502331-fda7c156-0df1-40ee-ba76-60fdded5e822.png)

**Comment by syalioune** [comment]
Hello @Adedamola86 
The test connection feature perform a call to the API GET /v1/project which requires VIEW_PORTFOLIO permission.
Adding that permission should resolve your issue.
